internal/oidc/token/token_handler.go

```diff
@@ -8,10 +8,13 @@ import (
 	"context"
```

Comment:
I did not look at the tests / other commits, but the changes to this file in commit 3301a62053a33da000d0af7996276e88b993521c look fine to me.

Reply:
Thanks. The other commits are only showing up because I branched off of the other PR to avoid merge conflicts. No need to review them because Margo reviewed that other PR already. Just a little shortcut taken so we can get to merging all these PRs faster without conflicts. :)